api-admin/src/main/java/bio/terra/pearl/api/admin/controller/siteContent/SiteContentController.java

```diff
@@ -35,6 +35,7 @@ public SiteContentController(
   public ResponseEntity<Object> get(
       String portalShortcode, String stableId, Integer version, String language) {
     if (StringUtils.isBlank(language)) {
+      // TODO: Switch this to use the default portal language
```

Comment:
I've left a bunch of TODO reminders in here. I intend to deal with them in an upcoming PR, not this PR